Cargo.toml

```diff
@@ -68,6 +68,10 @@ ipnet = { version = "2.7.0", features = ["serde"] }
 http-types = { version = "2.12.0", default-features = false }
 netns-rs = "0.1.0"
 textnonce = { version = "1.0.0" }
+boringssl = "0.0.5"
+kube = { version = "0.78.0", default-features = false, features = ["runtime","derive","rustls-tls", "client"] }
```

Comment:
can we turn off rustls

Reply:
We need to put something in features, otherwise it will use default one which is openssl.

Reply:
We can ONLY using `boring` for TLS. No openssl, rustls, etc. This is a hard requirement that no other libraries can do crypto operations.

Reply:
Updated, PTAL